### ISSUE TITLE
An img element with no alt attribute must not have any aria-* attributes other than aria-hidden .

### ISSUE THREAD

**Issue by robertmausser** [cgkineo/adapt-navigationLogo#7]
Error An img element with no alt attribute must not have any aria-*
attributes other than aria-hidden .
From line 42, column 32; to line 42, column 186
ion-logo"><img loading="eager" class="navigation-logo__image"
id="logo1" src="course/en/images/cse-logo-header.svg" arialabel="
Public Health Agency of Canada logo"><img l
5.

**Comment by oliverfoster** [comment]
We use aria-label rather than alt.
Based on the computation presedence from the Accessible Name and Description computation algorithm https://www.w3.org/TR/accname-1.2/#mapping_additional_nd

It says, aria-labelledby, aria-describedby, aria-label, then alt then title. Being that the alt attribute only works on the img tag and aria-label is supported in all screen readers on all elements, we use aria-label as its intent is more specific.
